### DISCUSSION TITLE
Support for Awesome Pages in integrated Blogs plugin

### DISCUSSION THREAD

**Discussion by Andre601** [squidfunk/mkdocs-material#5687] [Ideas]
It seems like the integrated blogs plugin doesn't support Awesome pages being used.

This is shown by the fact that the following error is printed on serve, despite a `index.md` file being provided and added in nav by the awesome pages plugin:
```
ERROR    -  Blog root 'posts/index.md' not in navigation.
```

Support for it would be nice.

**Comment by squidfunk** [comment]
[There's a PR on Insiders](https://github.com/squidfunk/mkdocs-material/discussions/5132#discussioncomment-5897535) that still needs feedback by users, that allows to omit the index file, so the blog plugin will just append it to the navigation. I haven't received any feedback on that and it still needs some work, but it is definitely planned.
